src/main/java/org/tndata/android/compass/fragment/LauncherFragment.java

```diff
@@ -9,9 +10,9 @@
 import android.view.View;
 import android.view.View.OnClickListener;
 import android.view.ViewGroup;
-import android.widget.ProgressBar;
 
 import org.tndata.android.compass.R;
+import org.tndata.android.compass.databinding.FragmentLauncherBinding;
```

Comment:
Heh, mine as well when I got started, and it is funny, every time I make a layout databinding compatible I gotta restart the IDE, rebuilding doesn't do the trick...